### PR TITLE
Fix : Add tag_filter type to NightwatchOptions interface. #3825

### PR DESCRIPTION
**Pull request by Ykumar1415**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: (https://github.com/nightwatchjs/nightwatch/issues/3825)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## nightwatch ([<kbd>unpkg</kbd>](https://unpkg.com/browse/nightwatch@latest/))
was missing the following properties:
1. createClient
1. client
1. cli
1. runTests
1. CliRunner

<details>
<summary>as well as these 2 other properties...</summary>
<p>initClient, runner</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0ab92714381a09e2f81ad8fe37131728fc4a820d
</p>

File: types/nightwatch/index.d.ts [hunk 1]
```diff
@@ -594,6 +594,12 @@ export interface NightwatchOptions {
      */
     skiptags?: string;
 
+	/**
```
Comment (DangerBotOSS) [inline types/nightwatch/index.d.ts:597]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

Comment (garg3133) [reply types/nightwatch/index.d.ts:597]: @Ykumar1415 Can you replace tabs with spaces here?

Comment (Ykumar1415) [reply types/nightwatch/index.d.ts]: sir is it fine now ?

File: types/nightwatch/index.d.ts [hunk 2]
```diff
@@ -594,6 +594,12 @@ export interface NightwatchOptions {
      */
     skiptags?: string;
 
+    /**
+	 * Tag(s) used/to be used during test execution.
+	 * Can be a single tag or an array of tags.
+	 */
+	tag_filter?: string | string[];
```
Comment (MysteryBlokHed) [inline types/nightwatch/index.d.ts:601]: It looks like there are a mix of spaces and tabs right now. Try making them all spaces:

```suggestion
    /**
     * Tag(s) used/to be used during test execution.
     * Can be a single tag or an array of tags.
     */
    tag_filter?: string | string[];
```

**Comment by garg3133** [comment]
Created https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66203 to fix this along with a test. Thanks @Ykumar1415 for your contribution.

**Pull request closed without merging**